modules/lvms-about-lvm-storage-installation.adoc

```diff
@@ -20,9 +20,11 @@ When using {lvms} on multi-node clusters, {lvms} only supports provisioning loca
 
 The prerequisites to install {lvms} are as follows: 
 
-* Before deploying {lvms}, ensure that every managed cluster has dedicated disks that are used to provision storage. {lvms} uses only those disks that are empty and do not contain file system signatures. To ensure that the disks are empty and do not contain file system signatures, wipe the disks before using them.
+* Ensure that you have a minimum of 10 millicpus and 100Mi of RAM.
```

Comment:
Some things on units here:

* `100Mi` is an incomplete unit. Is it mebibytes (MiB) or mebibits (Mib)? 
* Either way, add a space between the value `100` and the unit abbreviation (e.g. `100 MiB`).
* I am unfamiliar with "millicpus", but [Kubernetes usage](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#cpu-units) seems to be `10 milliCPU` in this case.

Reply:
Hi @jeana-redhat 
It is mebibytes. Updated it to `MiB`.
Thanks.

Reply:
Sorry, I thought I took care of this yesterday. Thanks for relabeling it to remind me :)